Support updates (#6797)
PostHog/posthog.com: contents/handbook/engineering/support-hero.md

Every week, we assign one person to be the "Support Hero" per engineering team (alternative names "Secondary", "Support Sidekick", "Infra Hero", "Luigi"). If this is you this week, congratulations! Support hero is an intense but super fun week where you get to talk to a bunch of users, get the satisfaction of helping them out, and contribute to a lot of different parts of our system. Your first priority should be dealing with alerts or Sentry alerts are high priority. After that, it's responding to customer support requests. Depending on how busy the week is you can do some feature work too.

> Marcus will act as triage for Product Analytics and Pipeline due to the high volume of tickets for those teams.  He will resolve tickets if possible, and escalate to the Engineering team responsible if he needs further help.

Reviewer: ```suggestion
[Marcus](https://posthog.com/community/profiles/1036), our Support Engineer, will triage tickets for the Product Analytics and Pipeline team, due to the high volume of tickets those teams get. He will resolve tickets if possible, and escalate to the Engineering team responsible if he needs further help.
```

- Start your response with `Hey [insert name], ...` and make sure you're polite, not everyone you talk to is an engineer and as accepting of terse messages

>   - If it's an email, make sure you format your message as an email and only send a single message, not multiple

Reviewer: ```suggestion
```
Since we do not have email support anymore, we can remove this line
